features/shop/cart_with_multiple_tax_rates.feature

```diff
@@ -0,0 +1,31 @@
+@ui-cart
+Feature: Apply correct tax for items with different tax rates
```

Comment:
I am fine with long file names, less vague = better. Feature files should be as decoupled as possible, with clear names and scenarios related to a single feature.

I'd rename ``cart_without_taxes`` to ``not_applying_taxes_for_products_without_tax_category.feature`` or something like that. Cart without taxes is not a feature. :D